cmd/backup.go

```diff
@@ -79,3 +81,12 @@ var backupCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func init() {
+	rootCmd.PersistentFlags().StringVar(
```

Comment:
Shouldnt this flag be used only with `backupCmd` or do we really need the config flag on every subcommand?

Reply:
Good remark, I will move it to the `backupCmd`!